chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/certifiedProduct/CertifiedProductValidatorImpl.java

```diff
@@ -99,6 +99,18 @@
     public CertifiedProductValidatorImpl() {
         urlRegex = Pattern.compile(URL_PATTERN);
     }
+    
+    public int getMaxLength(String field){
+    	return Integer.parseInt(String.format(
+    			messageSource.getMessage(new DefaultMessageSourceResolvable(field),
+    			LocaleContextHolder.getLocale())));
+    }
+    
+    public String getMaxLengthErrorMessage(String errorField){
+    		return String.format(
+    				messageSource.getMessage(new DefaultMessageSourceResolvable(errorField),
+    				LocaleContextHolder.getLocale()));
+    }
```

Comment:
These should be combined. All of the uses are of the form:
```
if value != null && check length of value is bad then
add error message (from this function)
```
The `checkMaxLength` function is never used without the `getMaxLengthMessage` function, so they should be combined, something like:
```
public checkLength(product, field, errorField)
if field != null && field.length > Integer.parseInt('maxlength.' + propertiesFile.errorField)
product.addErrorMessage(String.format...(getMessage('product.' + errrorField + 'length'))
``` 
up here, and in the checks below, something like (for the certification edition):
```
checkLength(product, product.certificationEdition, 'certificationEdition');
```